docs/development.md

```diff
@@ -0,0 +1,160 @@
+# Setting up Development Environment
+
+## Install Node.js
+
+Install Node.js by your favorite method, or use Node Version Manager by following directions at https://github.com/creationix/nvm
+
+```bash
+nvm install v4
+```
+
+## Fork and Download Repositories
+
+To develop bitcore-node:
+
+```bash
+cd ~
+git clone git@github.com:<yourusername>/bitcore-node.git
+git clone git@github.com:<yourusername>/bitcore-lib.git
+```
+
+To develop bitcoin or to compile from source:
+
+```bash
+git clone git@github.com:<yourusername>/bitcoin.git
+git fetch origin <branchname>:<branchname>
+git checkout <branchname>
+```
+**Note**: See bitcoin documentation for building bitcoin on your platform.
+
+
+## Install Development Dependencies
+
+For Ubuntu:
+```bash
+sudo apt-get install libzmq3-dev
+sudo apt-get install build-essential
+```
```

Comment:
these dependencies are necessary even if you aren't developing for bitcore-node. Also, if the user has libzmq-dev installed (libzmq1-dev), they will need to remove it: apt-get remove libzmq1-dev. I think the zmq install leverages pkg-config in order to find the zmq libs and pkg-config grabbed the wrong one for my install.